package.json

```diff
@@ -45,7 +45,7 @@
     "build": "grunt",
     "module": "git submodule init && git submodule update",
     "start": "source env/bin/activate && python run.py",
-    "virtualenv" : "pip install virtualenv && virtualenv env &&  source env/bin/activate",
+    "virtualenv" : "pip install virtualenv && virtualenv env &&  npm run pip && source env/bin/activate",
```

Comment:
I ran into issues with running `npm init` and `npm virtualenv`, because of the double install. 
Worked for me when removing `&& npm run pip` in `npm init`.

I also switched `npm init` to `npm setup` as discussed.